<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clientWidget</class>
 <widget class="QWidget" name="clientWidget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Панель администратора</string>
  </property>
  <widget class="QTabWidget" name="clientSelector">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>721</width>
     <height>481</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="foodsTab">
    <attribute name="title">
     <string>Наше меню</string>
    </attribute>
    <widget class="QTableWidget" name="foodsTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>711</width>
       <height>431</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>511</width>
       <height>0</height>
      </size>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="columnCount">
      <number>4</number>
     </property>
     <column>
      <property name="text">
       <string>Наименование</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Цена</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Свойства</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ингредиенты</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="placeOrderButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>430</y>
       <width>231</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Заказать еду</string>
     </property>
    </widget>
    <widget class="QLabel" name="orderStatusLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>430</y>
       <width>471</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="ordersTab">
    <attribute name="title">
     <string>Мои заказы</string>
    </attribute>
    <widget class="QTableWidget" name="myOrdersTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>711</width>
       <height>461</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Время заказа</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Клиент</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Сумма</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Статус оплаты</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Способ получения</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Статус</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Адрес доставки</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Телефон</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Время готовности</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Состав заказа</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="clientExitButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </widget>
  <widget class="QLabel" name="welcomingLabel">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>6</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <zorder>clientExitButton</zorder>
  <zorder>clientSelector</zorder>
  <zorder>welcomingLabel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
